### PR TITLE
Add drawing to hpp

### PR DESCRIPTION
**Pull request by doctormo**
This fills out the hpp interface a little more.

I'm going to send these to you progressively so I don't end up with a large delta.

### CODE REVIEW

File: include/capypdf.hpp [hunk 1]
```diff
@@ -122,20 +129,51 @@ class GraphicsState : public CapyC<CapyPDF_GraphicsState> {
...
 };
 
 class DrawContext : public CapyC<CapyPDF_DrawContext> {
     friend class Generator;
 
 public:
+    void cmd_b() { CAPY_CPP_CHECK(capy_dc_cmd_b(*this)); }
+    void cmd_B() { CAPY_CPP_CHECK(capy_dc_cmd_B(*this)); }
+    void cmd_Bstar() { CAPY_CPP_CHECK(capy_dc_cmd_Bstar(*this)); }
+    void cmd_bstar() { CAPY_CPP_CHECK(capy_dc_cmd_bstar(*this)); }
```
Comment (jpakkane) [inline include/capypdf.hpp:144]: The case order is different here. Lowercase versions should come before upper case ones for consistency.

File: include/capypdf.hpp [hunk 2]
```diff
@@ -122,20 +129,51 @@ class GraphicsState : public CapyC<CapyPDF_GraphicsState> {
...
+    void cmd_q() { CAPY_CPP_CHECK(capy_dc_cmd_q(*this)); }
+    void cmd_Q() { CAPY_CPP_CHECK(capy_dc_cmd_Q(*this)); }
+
     void cmd_re(double x, double y, double w, double h) {
         CAPY_CPP_CHECK(capy_dc_cmd_re(*this, x, y, w, h));
     }
     void cmd_rg(double r, double g, double b) { CAPY_CPP_CHECK(capy_dc_cmd_rg(*this, r, g, b)); }
 
+    void set_nonstroke(Color &color) { CAPY_CPP_CHECK(capy_dc_set_nonstroke(*this, color.get())); }
+    void set_stroke(Color &color) { CAPY_CPP_CHECK(capy_dc_set_stroke(*this, color.get())); }
```
Comment (jpakkane) [inline include/capypdf.hpp:164]: The `.get()` should not be needed here.

**Comment by doctormo** [comment]
Should be ready now

**Comment by jpakkane** [comment]
Nice, thanks.

**Pull request merged**